web/gds-user-ui/src/components/OrganizationProfile/TrisaDetail.tsx

```diff
@@ -12,75 +12,62 @@ const TrisaDetail: React.FC<TrisaDetailProps> = ({ data, type }) => {
     switch (data?.status) {
       case 'NO_VERIFICATION':
         return (
-          <Tag bg={'orange'} color={'white'} size={'sm'}>
```

Comment:
`md` is the default `size`. Removing `sm` increased the text's font size so that it will be consistent with other data displayed in the table.